tools/mypy_wrapper.py

```diff
@@ -202,12 +205,14 @@ def main(args: List[str]) -> None:
     and allows you to set the path to the mypy executable.
     """
     repo_root = str(Path.cwd())
-    exit_code, mypy_issues = run(
+    exit_code, mypy_issues, stderrs = run(
         args=[arg for arg in args if not arg.startswith(repo_root)],
         files=[arg for arg in args if arg.startswith(repo_root)],
     )
     for issue in mypy_issues:
         print(issue)
+    for stderr in stderrs:
+        print(stderr, end='', file=sys.stderr)
```

Comment:
curious why `end=''` instead of the default newlines

Reply:
because the stderr is captured directly from `mypy`, so it already contains all the newlines it's supposed to; omitting `end=''` would add an extra newline that isn't supposed to be there